taxonomies/ingredients.txt

```diff
@@ -26280,13 +26286,14 @@ nl:gemalen maïs
 pt:milho moído
 # openfoodfacts:https://world.openfoodfacts.org/ingredient/fr:maïs-moulu 28 products in 3 languages @2018-09-24
 
-#######################################################################################################################################
+################################################################################################################
 #
-#                         Rice
+#                                                 R I C E
 #
-#######################################################################################################################################
+#################################################################################################################
 # Rice is the seed of the grass species Oryza sativa (Asian rice) or Oryza glaberrima (African rice).
 
+#<en:cereal
```

Comment:
why is it commented?

Reply:
It was not in there. So I wondered whether it should/can be added.

Reply:
well if rice is a cereal, then I guess so.